src/components/intern-app.html

```diff
@@ -33,16 +33,19 @@
     <header>
       <user-list></user-list>
       <create-user></create-user>
-      <edit-user></edit-user>
+      <!-- <edit-user></edit-user> -->
     </header>
   </template>
 
   <script type="module">
     import { Element as PolymerElement } from '@banno/polymer/polymer-element.js';
+    import { LocalStorage } from '../api/localStorage.js';
+
+    const Storage = new LocalStorage();
```

Comment:
this should be set as a polymer prop and bound to all the components that need to use the `LocalStorage`